File: src/App/src/com/ss/stg/dto/CommentItem.java
package com.ss.stg.dto;

import java.util.Date;

public class CommentItem extends DTO {
	
	public CommentItem(){
	}
	
	private String commentId;
	private String commentUserName;
	private Date creatingTime;
	private String targetId;
	private String comment;

	public String getCommentId() {
		return commentId;
	}

	public void setCommentId(String commentId) {
		this.commentId = commentId;
	}

	public Date getCreatingTime() {
		return creatingTime;
	}

	public void setCreatingTime(Date creatingTime) {
		this.creatingTime = creatingTime;
	}

	public String getComment() {
		return comment;
	}

	public void setComment(String comment) {
		this.comment = comment;
	}

	public String getCommentUserName() {
		return commentUserName;
	}

	public void setCommentUserName(String commentUserName) {
		this.commentUserName = commentUserName;
	}

	public String getTargetId() {
		return targetId;
	}

	public void setTargetId(String targetId) {
		this.targetId = targetId;
	}
}

File: src/SextantTG/SextantTG.WebServices/DTO/CommentItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SextantTG.WebServices.DTO
{
    public class CommentItem : DTO
    {
        /// <summary>
        /// 评论ID
        /// </summary>
        public string CommentId { get; set; }

        /// <summary>
        /// 评论人
        /// </summary>
        public string CommentUserName { get; set; }

        public string UserId { get; set; }

        /// <summary>
        /// 创建时间
        /// </summary>
        public DateTime CreatingTime { get; set; }

        /// <summary>
        /// 被评论ID
        /// </summary>
        public string TargetId { get; set; }

        /// <summary>
        /// 评论
        /// </summary>
        public string Comment { get; set; }
    }
}